docs/manual/java/guide/broker/code/docs/javadsl/mb/AnotherServiceImpl.java

```diff
@@ -55,4 +62,60 @@ private void subscribeWithMetadata() {
         //#subscribe-to-topic-with-metadata
 
     }
+
+    private void skipMessages() {
+        //#subscribe-to-topic-skip-messages
+        helloService.greetingsTopic()
+            .subscribe()
+            .atLeastOnce(Flow.fromFunction((GreetingMessage message) -> {
+                if (message.getMessage().equals("Kia ora")) {
+                    return doSomethingWithTheMessage(message);
+                } else {
+                    // Skip all messages that do not start with "Hello"
+                    return Done.getInstance();
+                }
+            }));
+        //#subscribe-to-topic-skip-messages
+    }
+
+    //#subscribe-to-topic-partition-messages
```

Comment:
All this sample code is rather advanced for the average Lagom user. I think we should add a disclaimer in the code itself (in a comment, for example) explaining that this is advanced and that it assumes familiarity with akka-streams `Graph` DSL (and a link to https://doc.akka.io/docs/akka/current/stream/stream-graphs.html?language=scala so readers can either skip this bit or jump to akka docs).

Reply:
Sorry, I was ambiguous. Link in markdown is definitely better. :-D

Reply:
I've raised a pull request to fix all of the existing broken links in #1332.

Sorry, we don't currently automate this check (because it can break due to changes outside our control) so it requires a little bit of manual interpretation when you run it against a pull request.

Now that Travis supports cron-style jobs, it would be a good idea to run the check as a separate periodic build. Logged in #1333.